src/Compilers/CSharp/Portable/Symbols/SignatureOnlyParameterSymbol.cs

```diff
@@ -104,10 +110,8 @@ public override int GetHashCode()
             return Hash.Combine(
                 _type.GetHashCode(),
                 Hash.Combine(
-                    Hash.CombineValues(_customModifiers),
```

Comment:
I didn't understand why customer modifiers don't contribute to the hash code after this change.
Shouldn't we has both `_customModifiers` and `_refCustomModifiers`?